libflagship/mqttapi.py

```diff
@@ -1,14 +1,19 @@
 import paho.mqtt.client as mqtt
 import paho.mqtt
+from os import path
 import logging as log
 import ssl
 import json
 import uuid
 from datetime import datetime, timedelta
 
+from config import ROOT_DIR
```

Comment:
This is a problem - `libflagship` does not include anything except its own modules, and dependencies from `requirements.txt`, but now also this from a "foreign" directory (beyond top level of `libflagship/`).

This makes it impossible to refactor `libflagship` into a independent module, which is something we might want later.

Fortunately, the solution is super simple, see below :)

Reply:
✅